app/web/src/AppRoutes.tsx

```diff
@@ -208,23 +243,33 @@ export default function AppRoutes() {
       </AppRoute>
 
       {process.env.REACT_APP_IS_COMMUNITIES_PART2_ENABLED && (
-        <Switch>
-          <AppRoute isPrivate path={newPlaceRoute}>
-            <NewPlacePage />
-          </AppRoute>
-          <AppRoute isPrivate path={placeRoute}>
-            <PagePage pageType={PageType.PAGE_TYPE_PLACE} />
-          </AppRoute>
-          <AppRoute isPrivate path={newGuideRoute}>
-            <NewGuidePage />
-          </AppRoute>
-          <AppRoute isPrivate path={guideRoute}>
-            <PagePage pageType={PageType.PAGE_TYPE_GUIDE} />
-          </AppRoute>
-          <AppRoute isPrivate path={groupRoute}>
-            <GroupPage />
-          </AppRoute>
-        </Switch>
+        <Route
```

Comment:
I was wondering for a while why the 404 page didn't show up in dev. How does this extra route helps? Was it just not matching the `AppRoute` within this extra `Switch`?

Reply:
I'm not sure tbh. Seems like the last 404 rote was being ignored. Maybe `Switch` only renders the first thing that is valid, which was the nested `Switch`